src/main/java/seedu/nutrigoals/logic/commands/TipCommand.java

```diff
@@ -0,0 +1,26 @@
+package seedu.nutrigoals.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.nutrigoals.logic.commands.exceptions.CommandException;
+import seedu.nutrigoals.model.Model;
+
+
+/**
+ * Returns a fitness/dieting tip to the user.
+ */
+public class TipCommand extends Command {
```

Comment:
Might be a good idea to write some test cases for newly-added classes and methods like this one!